### ISSUE TITLE
Train model on full dataset

### ISSUE THREAD

**Issue by CatarinaPC** [flairNLP/flair#1457]
I have tested several word embeddings by training different models with my dataset split in train/dev/test.

Now that I have found the "best" word embeddings, I want to train a model on the full dataset.

I thought of putting the full dataset as the train, test and dev. But won't this cause trouble with dev in particular?

**Comment by djstrong** [comment]
There will be problem with early stopping.

**Comment by CatarinaPC** [comment]
I'm experimenting with using train_with_dev=True, so this way I can use the full dataset for training. Do you think this is a viable and correct way of accomplishing my goal?

**Comment by djstrong** [comment]
Then the training loss will be minimized so the model can overfit.
There is no universal solution for the problem.
